From a139c403e63a47b53b29df9588bed66db3d0d8d1 Mon Sep 17 00:00:00 2001
From: Chris Leech <cleech@redhat.com>
Date: Mon, 25 Nov 2013 22:28:12 -0800
Subject: [PATCH] start socket listeners on iscsiadm command

fix for trying to run iscsiadm commands right after installing the rpm
without manually starting the systemd units
---
 etc/iscsid.conf | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/etc/iscsid.conf b/etc/iscsid.conf
index db395df..16c00f0 100644
--- a/etc/iscsid.conf
+++ b/etc/iscsid.conf
@@ -17,7 +17,8 @@
 # maintainers.
 #
 # Default for Fedora and RHEL. (uncomment to activate).
-#iscsid.startup = /bin/systemctl start iscsid.service
+# Use socket activation, but try to make sure the socket units are listening
+iscsid.startup = /bin/systemctl start iscsid.socket iscsiuio.socket
 # 
 # Default for upstream open-iscsi scripts (uncomment to activate).
 # iscsid.startup = /sbin/iscsid
-- 
2.1.0

